.github/workflows/format.yml

```diff
@@ -3,8 +3,13 @@ name: Format the code
 on:
   workflow_dispatch:
 
+permissions: {}
 jobs:
   createPullRequest:
+    permissions:
+      contents: write # to create branch (peter-evans/create-pull-request)
```

Comment:
What's this peter-evans comment?

Reply:
It is an identifier of the action [that is used in the workflow](https://github.com/keras-team/keras/blob/2eda0600e24c48363a8eb07ad2c3f39436430df7/.github/workflows/format.yml#L32).